src/util/specific.py

```diff
@@ -137,3 +139,81 @@ def reset_cowed_views(cr, xmlid, key=None):
         [key, module, name],
     )
     return set(sum(cr.fetchall(), ()))
+
+
+def translation2jsonb_all_missing(cr):
+    """Convert all missing translated fields to jsonb.
+
+    This helper function is designed to run only in an on-premise Odoo instance
+    where any unofficial modules you use are installed, after going through the
+    upgrade to version 16.0.
+
+    It will find all unofficial fields and convert translations from the
+    ``_ir_translation`` table into the new JSONB format.
+
+    Modules must be available in the environment to be loaded.
+
+    :return: Converted fields.
+    """
+    if not version_gte("16.0"):
+        raise UpgradeError("JSONB translations are only available from Odoo 16")
```

Comment:
Handle the case where the `_ir_translation` has been removed (once all fields have been converted, we can drop this table).
```suggestion
    if not version_gte("16.0"):
        raise UpgradeError("JSONB translations are only available from Odoo 16")
    if not table_exists(cr, "_ir_translation"):
        _logger.getChild("translation2jsonb_all_missing").info("Missing `_ir_translation` table; skip.")
        return
```